tidy3d/plugins/microwave/custom_path_integrals.py

```diff
@@ -123,8 +97,8 @@ def compute_integral(
         field2_interp = field2.interp(path_indexer, method="linear")
 
         # Determine the differential length elements along the path
-        dl_x = self._compute_dl_component(x_path, self.is_closed_contour)
-        dl_y = self._compute_dl_component(y_path, self.is_closed_contour)
+        dl_x = CustomPathIntegral2DSpec._compute_dl_component(x_path, self.is_closed_contour)
+        dl_y = CustomPathIntegral2DSpec._compute_dl_component(y_path, self.is_closed_contour)
```

Comment:
for closed curves first and last points are the same, are we accounting twice the same point when performing integration here? should one of the points be excluded?

Reply:
ah, I missed that you later do `.integrate()` which should take care of that. Ignore this comment